tests/query.test.js

```diff
@@ -0,0 +1,17 @@
+const query = require("../src/resolvers/Query");
+const { getPrismaTestInstance } = require("./init/prismaTestInstance");
+
+const parent = {};
+const ctx = {
+    prisma: getPrismaTestInstance()
+};
+
```

Comment:
In order to test the query functions it will need consistent data that it can query against.  You can create a notification and have it run before your test by using Jest's `beforeAll()` function or in your test block before running `expect()`.